apps/test-ui/src/TestSessionView.tsx

```
@@ -32,7 +32,7 @@
 // =============================================================================
 
 import React, { useState, useEffect, useCallback, useRef } from 'react';
-import { useShellConnection } from 'shell-ui';
+import { useShellConnection } from 'shell';
```

ESLint

[error] 'shell' import is restricted from being used. Renamed: import from 'shell'.

(no-restricted-imports)